.github/ISSUE_TEMPLATE/infrastructure.yml

```diff
@@ -0,0 +1,31 @@
+name: ⚙️ Infrastructure Issue
+description: File an issue relating to infrastructure or automation
+title: "[Infrastructure]: <title>"
+labels: ["infrastructure", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        The following fields are required:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
```

Comment:
Change to "Please search to see if an issue already exists covering this"
(There's no bug here)

Reply:
Sounds good. To address point nr. 2 I think it makes sense to search for an _open_ issue. At this point I think it's better not to re-open past closed issues due to the number of recent changes.